cronjobs/ezplatformindexsubtree.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+if ( !$isQuiet )
+{
+    $cli->output( "Processing pending subtree re-index actions" );
+}
+
+// check that solr is enabled and used
+$eZSolr = eZSearch::getEngine();
```

Comment:
Variable name should be changed, togethercwith comment above not to reference solr :)